docs/docs/api/index.mdx

```diff
@@ -6,13 +6,13 @@ sidebar_position: 10
 
 {/* Note: these contents are new and should be reviewed. The current website's API docs landing page has no description about what the packages do / when to use them. It's not very useful. */}
 
-To get started with the Fluid Framework, you'll want to take a dependency on our client library, [fluid-framework](./fluid-framework.md).
+To get started with the Fluid Framework, you'll want to take a dependency on our client library, [fluid-framework](./fluid-framework/index.md).
```

Comment:
We have to specify `index` here because we are expressing these links via file paths rather than routes. This is the recommended pattern.